composer.json

```diff
@@ -189,6 +190,7 @@
         "drupal/key": "^1.14",
         "drupal/layout_builder_component_attributes": "^1.1",
         "drupal/layout_builder_modal": "^1.1",
+        "drupal/layout_paragraphs": "^1.0@beta",
```

Comment:
Giving this a go eh? I am interested to hear about how it went at a later date.

Reply:
yeah we're gonna experiment with it some to see how it does :smile: